doc/newsfragments/2278_new.fast_kill.rst

```diff
@@ -0,0 +1,4 @@
+In interactive mode when testplan is stopped with ctr-c, instead of properly shutting down the tests, which can take
+even minutes for a complex environment, now a fast kill approach is killing all started test processes forcefully.
+The same happens when a testplan script is interrupted by ctr-c, or with the SIGINT signal. SIGTERM will shutdown the
```

Comment:
This is against our common sense though, ctrl-c is generating sigint, and:

SIGINT is a gentle nudge asking the process to stop. If that fails, SIGTERM is a more forceful, but still polite request for the process to ...

Reply:
SIGINT is asking the process to interrupt on what it is doing (and most of the time wait for new inputs) SIGTERM is asking the process to stop, both of that can be handled even ignored by the app.  in python SIGINT is interpreted as keyboard interrupt, because ctrl-c fire that signal. In this approach the user action of pressing ctrl-c is interpreted as there is an inpatient someone wan to stop this interactive or batch run so just do it fast. I found it reasonable. The user story we want to enable here is mostly manual runs where the user want to stop the interactive session and restart as fast as possible, the same apply for the batch run if it is interrupted by ctrl-c the user just want to stop it as he was seeing some problem in the logs, or realized something why it need to restart it.